release-notes/6.0/cve.md

```diff
@@ -7,7 +7,12 @@ Your app needs to be on the latest .NET 6 patch version to be secure. The longer
 ## Which CVEs apply to my app?
 
 Your app may be vulnerable to the following published security [CVEs](https://www.cve.org/) if you are using the given version or older.
-
+- 6.0.21 (August 2023)
+  - [CVE-2023-36799 | .NET Remote Code Execution Vulnerability](https://github.com/dotnet/announcements/issues/275)
+  - [CVE-2023-36796 | .NET Remote Code Execution Vulnerability](https://github.com/dotnet/announcements/issues/274)
+  - [CVE-2023-36793 | .NET Elevation of Privilege Vulnerability](https://github.com/dotnet/announcements/issues/273)
```

Comment:
Following this link takes me to "Microsoft Security Advisory CVE-2023-36793: .NET Remote Code Execution Vulnerability". That leaves me confused. Is this a Remote Code Execution or a Elevation of Privilege?

Reply:
It’s a typo. I will correct it in a new PR